### PR TITLE
Frontend, workflows and Docker files

### PR DESCRIPTION
**Pull request by SamanthaCra**
Documentation:\
https://github.com/OtagoPolytechnic/Cloudy-with-a-Chance-of-LoRa/wiki/App-Server-(Frontend)\
https://github.com/OtagoPolytechnic/Cloudy-with-a-Chance-of-LoRa/wiki/Docker

This has the DockerFile and some adjustments for our app once it goes live. 

The aim of Docker is to use the YAML to trigger a deploy on a pull to the main branch. So far app deployment and docker are seperate and will need to be developed further.

### CODE REVIEW

**Comment by SamanthaCra** [comment]
.gitignore and .dockerignore were also added.

**Review by fuzzyers — APPROVED** [review]
Secrets library looks good. Might be a good idea to add code comments in the yaml run scripts to increase readibility especially for the run's that are doing quite alot.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,11 @@
 # Cloudy-with-a-Chance-of-LoRa
 2024 S2 Studio 5/6 Weather Station Team
+
+We are using the LoRaWan/The Things Network to create a weather station that transmits data. 
+
+Team members:\
+Kieran Black\
```
Comment (kierenblack) [inline README.md:7]: Kieren Black

**Review by kierenblack — APPROVED** [review]
Apart from the one spelling mistake it all looks good.

**Pull request merged**